pkgs/create-neon/data/templates/ts/index.cts.hbs

```diff
@@ -0,0 +1,7 @@
+const addon = require('./load.cjs');
```

Comment:
```ts
declare module "./load.cjs" {
    function hello(): string;
}
```

This could also be done with a `.d.ts`. There is a trade-off between being "clean" and keeping the directory structure simple.

Reply:
Yeah I like this a lot, it tries to nudge people away from the any type. I think it's best to keep it in file for discoverability, and users can choose to refactor it out if it grows.